go.mod

```diff
@@ -115,7 +115,7 @@ require (
 	go.uber.org/multierr v1.1.0 // indirect
 	go.uber.org/zap v1.10.0
 	golang.org/x/crypto v0.0.0-20190611184440-5c40567a22f8
-	golang.org/x/net v0.0.0-20190628185345-da137c7871d7 // indirect
+	golang.org/x/net v0.0.0-20190628185345-da137c7871d7
```

Comment:
unable to get past thru linter checker on CI with `// indirect', so had to push without it just to get pass thru build successfully on CI

Reply:
Do you know why it was complaining ?